lighthouse-core/gather/gatherers/dobetterweb/stacks.js

```diff
@@ -8,22 +8,69 @@
  * @fileoverview Gathers a list of detected JS libraries and their versions.
  */
 
+/* global window */
+/* global d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests */
+
 'use strict';
 
-const JSLibraries = require('./js-libraries');
+const Gatherer = require('../gatherer');
+const fs = require('fs');
+const libDetectorSource = fs.readFileSync(
+  require.resolve('js-library-detector/library/libraries.js'), 'utf8');
+
+/**
+ * @typedef JSLibrary
+ * @property {string} name
+ * @property {string} version
+ * @property {string} npmPkgName
+ * @property {string} iconName
+ */
 
-class Stacks extends JSLibraries {
+/**
+ * Obtains a list of detected JS libraries and their versions.
+ */
+/* istanbul ignore next */
+function detectLibraries() {
+  /** @type {JSLibrary[]} */
+  const libraries = [];
+
+  // d41d8cd98f00b204e9800998ecf8427e_ is a consistent prefix used by the detect libraries
+  // see https://github.com/HTTPArchive/httparchive/issues/77#issuecomment-291320900
+  // @ts-ignore - injected libDetectorSource var
+  Object.entries(d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests).forEach(async ([name, lib]) => { // eslint-disable-line max-len
+    try {
+      const result = await lib.test(window);
+      if (result) {
+        libraries.push({
+          name: name,
+          version: result.version,
+          npmPkgName: lib.npm,
```

Comment:
since you're renaming stuff to `npm` then might as well keep this `npm` between here and L76